src/matrix/matrix-common.h

```diff
@@ -24,12 +24,11 @@
 // files in this directory.
 
 #include "base/kaldi-common.h"
-#include "matrix/kaldi-blas.h"
 
 namespace kaldi {
 typedef enum {
```

Comment:
there needs to be some comment to the effect that these are the same as CblasTrans and CblasNoTranss from the CBLAS library, but we are writing them as numbers because we don't include that library in these headers.

Reply:
done

Reply:
1) CLAPACK - well compiling, but when linking there is an error: undefined reference to 'cblas_*'. I think error in src/configure, because no linking blas library is specified, for instance when linking align-egual. And I suppose CBLAS should be istalled independtly.
2) OpenBLAS - well compiling and linking. (But there is difficulties, which I fixed) There are many many warnings 'linker input file unused because linking not done'. Matrix-lib-text is passed. But we need include cblas-wrappers.h into matrix-lib-test.cc.
3) mkl - tomorrow